browser-test/src/applicant/application_address_correction.test.ts

```diff
@@ -226,6 +245,18 @@ test.describe('address correction', () => {
       await logout(page)
     })
 
+    test('skips address correction if optional address question is not answered', async () => {
+      const {page, applicantQuestions} = ctx
+      await enableFeatureFlag(page, 'esri_address_correction_enabled')
+      await applicantQuestions.applyProgram(optionalAddressProgram)
+
+      await applicantQuestions.clickNext()
+
+      await applicantQuestions.expectReviewPage()
+      await applicantQuestions.clickSubmit()
+      await logout(page)
```

Comment:
(fyi) Many of our older browser tests (still) use shared database state within the a single test file, that is it clears the database once when the file starts tests, but not between each test. You can tell if it's got
`const ctx = createTestContext(/* clearDb= */ false)` set with `false`. It's possible you may find if you run a single test in the file there may be a test that expects the database to be in a state from a change in another test.

Reply:
Ack good to know!